packages/next-tinacms-github/src/get-github-static-props.ts

```diff
@@ -0,0 +1,70 @@
+/**
+
+Copyright 2019 Forestry.io Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+import { GithubError } from './github/content/GithubError'
+import { getMarkdownFile } from './github/content/getMarkdownFile'
+import { getJsonFile } from './github/content/getJsonFile'
+
+export interface PreviewMeta {
+  github_access_token: string
```

Comment:
Should these be camelCase?

Reply:
I would be up for that. Those variable names from [react-tinacms-github](https://github.com/tinacms/tinacms/blob/master/packages/next-tinacms-github/src/constants.ts#L22) though so that change feels out of scope for now.